cli/formatting.go

```diff
@@ -0,0 +1,192 @@
+			),
+		)
+	}
+	return columnize.Format(output, config)
+}
+
+func FormatMemory(account *eos.AccountResp, config *columnize.Config) string {
+	output := []string{
+		"memory:",
+		fmt.Sprintf("     quota: %s| used: %s",
+			prettifyBytes(account.RAMQuota),
+			prettifyBytes(account.RAMUsage),
+		),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatNetworkBandwidth(account *eos.AccountResp, config *columnize.Config) string {
+	delegatedNet := account.TotalResources.NetWeight.Sub(account.SelfDelegatedBandwidth.NetWeight)
+
+	output := []string{
+		"net bandwidth:",
+		fmt.Sprintf("     staked:|%s|(total stake delegated from account to self)",
+			prettifyAsset(account.SelfDelegatedBandwidth.NetWeight),
+		),
+		fmt.Sprintf("     delegated:|%s|(total stake delegated to account from others)",
+			prettifyAsset(delegatedNet),
+		),
+		fmt.Sprintf("     used:|%s", prettifyBytes(int64(account.NetLimit.Used))),
+		fmt.Sprintf("     available:|%s", prettifyBytes(int64(account.NetLimit.Available))),
+		fmt.Sprintf("     limit:|%s", prettifyBytes(int64(account.NetLimit.Max))),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatCPUBandwidth(account *eos.AccountResp, config *columnize.Config) string {
+	delegatedCPU := account.TotalResources.CPUWeight.Sub(account.SelfDelegatedBandwidth.CPUWeight)
+
+	output := []string{
+		"cpu bandwidth:",
+		fmt.Sprintf("     staked:|%s|(total stake delegated from account to self)",
+			prettifyAsset(account.SelfDelegatedBandwidth.CPUWeight),
+		),
+		fmt.Sprintf("     delegated:|%s|(total stake delegated from account from others)",
+			prettifyAsset(delegatedCPU),
+		),
+		fmt.Sprintf("     used:|%s", prettifyTime(int64(account.CPULimit.Used))),
+		fmt.Sprintf("     available:|%s", prettifyTime(int64(account.CPULimit.Available))),
+		fmt.Sprintf("     limit:|%s", prettifyTime(int64(account.CPULimit.Max))),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatBalances(account *eos.AccountResp, config *columnize.Config) string {
+	totalStaked := account.SelfDelegatedBandwidth.NetWeight.Add(account.SelfDelegatedBandwidth.CPUWeight)
+	totalUnstaking := eos.Asset{
+		Amount: 0,
+		Symbol: account.CoreLiquidBalance.Symbol,
+	}
+	if account.RefundRequest != nil {
+		totalUnstaking = account.RefundRequest.CPUAmount.Add(account.RefundRequest.NetAmount)
+	}
+	total := totalUnstaking.Add(totalStaked).Add(account.CoreLiquidBalance)
+
+	output := []string{
+		fmt.Sprintf("%s balances:", account.CoreLiquidBalance.Symbol.Symbol),
+		fmt.Sprintf("     liquid:|%s", prettifyAsset(account.CoreLiquidBalance)),
+		fmt.Sprintf("     staked:|%s", prettifyAsset(totalStaked)),
+		fmt.Sprintf("     unstaking:|%s", prettifyAsset(totalUnstaking)),
+		fmt.Sprintf("     total:|%s", prettifyAsset(total)),
+	}
+
+	return columnize.Format(output, config)
+}
+
+func FormatProducers(account *eos.AccountResp, config *columnize.Config) string {
+	output := []string{
+		"producers:",

```

Comment:
I think it would be clearer with `votes:` or `voted for:`, what do you think?

Reply:
I agree, `voted for:` reads better

Reply:
Updated